common/src/main/java/com/wynntils/services/mapdata/providers/builtin/CategoriesProvider.java

```diff
@@ -78,12 +80,12 @@ public int getPriority() {
 
                 @Override
                 public MapVisibility getIconVisibility() {
-                    return DEFAULT_VISIBILITY;
+                    return DEFAULT_ICON_VISIBILITY;
```

Comment:
```suggestion
                    return MapVisibility.DEFAULT_ICON_VISIBILITY;
```

etc, and then you can remove the static re-definition in this class.